lib/has_configuration/configuration.rb

```diff
@@ -51,7 +50,8 @@ def filename
         filename = "#{@class_name.downcase}.yml"
         defined?(Rails) ? Rails.root.join('config', filename).to_s : filename
       else
-        raise ArgumentError, "Unable to resolve filename, please add :file parameter to has_configuration"
+        fail ArgumentError,
+             'Unable to resolve filename, please add :file parameter to has_configuration'
```

Comment:
Line is too long. [90/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.